appengine/go11x/static/public/style.css

```diff
@@ -0,0 +1,4 @@
+body {
```

Comment:
I think the CSS and HTML files should have licenses. Maybe app.yaml doesn't support it? If other files don't have the header, we should add it.

Reply:
Done.